vsts/sdl.yaml

```diff
@@ -34,26 +34,42 @@ phases:
   - task: securedevelopmentteam.vss-secure-development-tools.build-task-vulnerabilityassessment.VulnerabilityAssessment@0
     displayName: 'Vulnerability Assessment'
 
-  - task: fortifyvsts.hpe-security-fortify-vsts.build-task-fortify-sca.FortifySCA@1
-    displayName: Fortify
+    ## Spotbugs doesn't have a false positive list, but instead recommends annotating the lines of code in question
+    ## with annotations like @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
+  - powershell: ./vsts/spotbugs.ps1
+    displayName: 'Run Spotbugs'
+
+    ## Semmle doesn't have a false positive list, but instead recommends marking the lines of code in question
+    ## with "// lgtm"
+    ## https://help.semmle.com/lgtm-enterprise/user/help/alert-suppression.html#suppresswarnings-annotation
+  - task: Semmle@1
+    #Only run this step if it isn't a pull request. Semmle only needs to run nightly
+    condition: ne(variables['Build.Reason'], 'PullRequest')
```

Comment:
SDL requirements are to run spotbugs at the gate, and both spotbugs and Semmle nightly